knowledge/highways/gaps_in_interstate/qna.yaml

```diff
@@ -0,0 +1,21 @@
+created_by: 
+domain: geography
+seed_examples:
+- answer: |
+    True gap is part of a numbered highway that is not physically continuous. They might be
+    Connected by another number road in between.
+  question: What are true gaps?
```

Comment:
Hey there, thank you so much for your submission! To help the process a bit, it is important, particularly with knowledge, to be as specific as possible. In this case we can see in the precheck that the model thinks the true gap refers to a math principal. Maybe try "what is a true gap in the context of Interstate Highways"